README.md

```diff
@@ -32,5 +36,14 @@ Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 The page will reload if you make edits.<br>
 You will also see any lint errors in the console.
 
+## Useful Links
+- [Contenta](http://www.contentacms.org/)
+- [Sample Source Data](http://live-contentacms.pantheonsite.io/)
+- [Sample Source Data APIs](http://live-contentacms.pantheonsite.io/api)
+- [Demo Implementation Guidelines](https://github.com/contentacms/contenta_jsonapi/blob/8.x-1.x/docs/demos-consumers-implementation-guidelines.md)
```

Comment:
I think this is the good link: https://contentacms.readthedocs.io/en/latest/demos-consumers-implementation-guidelines/